### ISSUE TITLE
Run two vagrant boxes simultaniously

### ISSUE THREAD

**Issue by tomjanssens** [joomlatools/joomlatools-vagrant#37]
At the moment it is not possible to run two vagrant boxes at the same time.

That’s because when running `vagrant up` (second step) vagrant tries to rename a temp folder located in user/VirtualBox VMs/ to joomlatools-box which already exists.

Note: you can solve this by just allowing the box to have a different name. We solved this for nooku-server box too. See: https://github.com/nooku/nooku-server/blob/develop/Vagrantfile.pkg#L30-L39 (see all commits that mention ticket #281)

**Comment by yiendos** [comment]
+1 from me, would make the development process much simpler.

**Comment by johanjanssens** [comment]
@stevenrombauts Something to add to the new box you are building?

**Comment by yiendos** [comment]
Hi @johanjanssens , I thought this might be a solution to a problem I've been having of developing the next vagrant box features but still having a `current version` of the box on my system at the same time. 

I discussed this with @stevenrombauts today and we came up with the idea of using a custom.yaml file: 
https://github.com/joomlatools/joomlatools-vagrant/blob/master/config.custom.yaml-dist

to extend the Vagrantfile:
https://github.com/joomlatools/joomlatools-vagrant/blob/master/Vagrantfile#L25

Namely with a config item for box name: 
https://github.com/joomlatools/joomlatools-vagrant/blob/master/Vagrantfile#L43

and box ip:
https://github.com/joomlatools/joomlatools-vagrant/blob/master/Vagrantfile#L38

This way two versions of the box can be run on the host system at the same time. 

I agree with steven that the box is already configured for multiple sites. Did you have a different use case in mind @johanjanssens ?
